lib/iex/lib/iex/pry.ex

```diff
@@ -36,7 +36,7 @@ defmodule IEx.Pry do
     opts = [
       binding: binding,
       dot_iex_path: "",
-      env: env,
+      env: %{env | tracers: []},
```

Comment:
```suggestion
      # Remove all tracers because the tracer code is most
      # likely stale by the time we are prying the code.
      env: %{env | tracers: []},
```